### DISCUSSION TITLE
Bluescreen in Windows 10 after wakeup while trying to re-establish sftp connection

### DISCUSSION THREAD

**Discussion by K9-711** [eduardofilo/RG350_adam_image#105] [General]
RG350M connected by USB to Windows 10. SFTP connection with FileZilla. Put RG into sleep mode. Trying to switch folder in Filezilla session and realize that RG is still in sleep mode. Wake up RG and try switch to another folder in Filezilla and Win10 shows a bluescreen and will reboot. Happened two times. 

Error: The computer has rebooted from a bugcheck.  The bugcheck was: 0x00000139 (0x0000000000000003, 0xffff8e032c46f5c0, 0xffff8e032c46f518, 0x0000000000000000). A dump was saved in: C:\WINDOWS\MEMORY.DMP. Report Id: 221f29c2-8f52-4f0d-a4eb-2376d951b9f5.

**Comment by eduardofilo** [comment]
I am sorry, but I can't work in this issue. It is a system topic. I only can say you that try with other mode for USB connection using the `USB Mode` entry in Apps section.
